shared/django_apps/reports/tests/factories.py

```diff
@@ -103,3 +104,31 @@ class Meta:
             ReportResults.ReportResultsStates.COMPLETED,
         ]
     )
+
+
+class TestFactory(DjangoModelFactory):
+    class Meta:
+        model = Test
+    
+    repository = factory.SubFactory(RepositoryFactory)
+    id = factory.Sequence(lambda n: f"test_{n}")
+
```

Comment:
It looks like the formatter wasn't run. More context: https://github.com/codecov/shared/pull/252#discussion_r1646379852 but please run the formatter manually.